lib/ts/recipe/oauth2provider/types.ts

```diff
@@ -349,6 +349,11 @@ export type RecipeInterface = {
         tenantId: string;
         userContext: UserContext;
     }): Promise<JSONObject>;
+    revokeToken(input: {
+        authorizationHeader?: string;
```

Comment:
Does this work with clientId + clientSecret in the body?

Reply:
Updated this to take both `authorizationHeader` and `clientId` + `clientSecret`